test/fuzz/gen.ml

```diff
@@ -136,9 +136,10 @@ let expr_available_2_f64 =
 let if_else expr ~locals ~stack env =
```

Comment:
You should add this to the list of blocks. It's valid to branch to an `if`, see:

```wast
(module

  (func $f
    i32.const 1
    (if $a (result i32)
      (then
        i32.const 42
        br $a
      )
      (else
        i32.const 999
      )
    )
    drop
  ))
  ```